### PR TITLE
Test changes 1

### PR DESCRIPTION
**Pull request by steven-vu1**

### CODE REVIEW

File: TIM-BC-Tool-SQCG/app/scripts/views/SegmentInfoView.js [hunk 1]
```diff
@@ -402,7 +402,7 @@ define([
 
         if( valid ) {
           if( value > 240 ) {
-            alert('Average Incident Duration Error: Should not exceed 240.\nPrevious valid value will be reloaded.');
```
Comment (dan-du-car) [inline TIM-BC-Tool-SQCG/app/scripts/views/SegmentInfoView.js:405]: The check condition is greater than 240. So it makes sense to say not exceed 240.

Comment (yingyanlou) [reply TIM-BC-Tool-SQCG/app/scripts/views/SegmentInfoView.js:405]: I agree with Dan.  Please restore the original alert.  In fact, the tool tips should be say 240 as well. This applies to all modules other than SSP (SSP's logic is a bit different and Dan is / will be investigating).  I have left detailed comments on what to say in the 0428 test report.

File: TIM-BC-Tool-DC/app/scripts/partials/SegmentInfo.html [hunk 2]
```diff
@@ -317,7 +317,7 @@ <h3 class="panel-title">Traffic Information</h3>
 				    			<th>WEATHER
 				    			<!--Added by Fang Zhou for adding speed reduction info for weather 09/22/2015-->
 				    				<a href="#" class="segment-info-helper" tabindex="1" data-toggle="tooltip" data-placement="top"
-								              title="The following percentage reductions are applied to the free low speed in the calculation:
+								              title="The following percentage reductions are applied to the free flow speed in the calculation:
```
Comment (yingyanlou) [inline TIM-BC-Tool-DC/app/scripts/partials/SegmentInfo.html:320]: Also left you comment on this in the 0428 test report.

**Review by yingyanlou — CHANGES_REQUESTED** [review]
Please address the comments, and incorporate additional changes requested in the 0428 test report.

**Pull request merged**